src/app/views/App.tsx

```diff
@@ -376,7 +389,7 @@ class App extends Component<IAppProps, IAppState> {
     let sidebarWidth = classes.sidebar;
     let layout = '';
     let sideWidth = sidebar.width;
-    let sideHeight = sidebar.height;
+    const sideHeight = '';
```

Comment:
Is this being used anywhere?

Reply:
Yes, on the resizable component.
![image](https://user-images.githubusercontent.com/45680252/162967259-201ca6bf-9470-4b5c-aca7-fcf4487acd08.png)
The value however doesn't change. Let me remove it and pass an empty string to the height property